build_tarballs.jl

```diff
@@ -3,60 +3,78 @@
 using BinaryBuilder
 
 name = "LibCURL"
-version = v"7.61.0"
+version = v"7.64.0"
 
 # Collection of sources required to build LibCURL
 sources = [
-    "https://curl.haxx.se/download/curl-7.61.0.tar.gz" =>
-    "64141f0db4945268a21b490d58806b97c615d3d0c75bf8c335bbe0efd13b45b5",
-
+    "https://curl.haxx.se/download/curl-7.64.0.tar.gz" =>
+    "cb90d2eb74d4e358c1ed1489f8e3af96b50ea4374ad71f143fa4595e998d81b5",
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/curl-7.61.0
-./configure --prefix=$prefix --host=$target --with-mbedtls --without-ssl --disable-manual
+cd $WORKSPACE/srcdir/curl-7.64.0
+
+# Set up ca path/bundle
+ca_args=""
+if [[ $target == x86_64-apple-darwin14 ]]; then
+    ca_args="--with-ca-bundle=/etc/ssl/cert.pem"
+elif [[ $target == *-linux-* ]]; then
+    ca_args="--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt"
+fi
```

Comment:
Just for testing? I believe you mentioned out-of-band that this just references a path and doesn't actually embed the certificate?

Reply:
yeah, I think it'll just put that path into the build so that it looks for the cacert there. I don't think there's a way to actually embed a cacert into it based on what I've seen so far. I could change this build to simply update everything, and perhaps mess with the certs at a later date, so we don't introduce this change into this PR.